lib/web_ui/lib/src/engine/platform_dispatcher.dart

```diff
@@ -770,7 +770,7 @@ class EnginePlatformDispatcher extends ui.PlatformDispatcher {
     call `updateSemantics`.
   ''')
   void updateSemantics(ui.SemanticsUpdate update) {
-    EngineSemanticsOwner.instance.updateSemantics(update);
+    implicitView?.semantics.updateSemantics(update);
```

Comment:
At some point, we need the framework to pass a `view` parameter here, right?

Reply:
According to the deprecation message above this method is going away forever.